packages/desktop/pages/_app.tsx

```diff
@@ -27,13 +28,28 @@ const client = new ApolloClient({
   cache: new InMemoryCache({
     typePolicies: {
       CombatUnitStub: {
-        keyFields: ['id', 'spec'],
+        keyFields: false,
```

Comment:
intentional?

Reply:
Yeah -- these shouldn't be cached by Apollo as individual items at all. We can't compute an appropriate key for the cache without doing something crazy like hashing their entire talents/equipment. There's also no perf hit here because we aren't ever using these in other queries or querying for them individually.